docs/developing/install.rst

```diff
@@ -12,255 +12,210 @@ environment the first thing you need to do is install h's system dependencies.
 
-    pip install -U pip tox>=3.8.0
+* `Docker CE <https://docs.docker.com/install/>`_ and `Docker Compose <https://docs.docker.com/compose/>`_.
+  Follow the `instructions on the Docker website <https://docs.docker.com/compose/install/>`_
+  to install these.
 
 
-Getting the h source code from GitHub
--------------------------------------
+* `pyenv`_.
+  Follow the instructions in the pyenv README to install it.
 
-Use ``git`` to download the h source code:
+Clone the Git repo
+------------------
 
-.. code-block:: bash
+.. code-block:: shell
 
-    git clone https://github.com/hypothesis/h.git
+   git clone https://github.com/hypothesis/h.git
 
 This will download the code into an ``h`` directory in your current working
-directory.
-
-Change into the ``h`` directory from the remainder of the installation
-process:
+directory. You need to be in the ``h`` directory from the remainder of the
+installation process:
 
-.. code-block:: bash
+.. code-block:: shell
 
    cd h
 
+Run the services with Docker Compose
+------------------------------------
 
-Installing the services
------------------------
+Install and run the services the h requires using Docker Compose:
 
-h requires the following external services:
+.. code-block:: shell
 
-- PostgreSQL_ 9.4+
-- Elasticsearch_ v6, with the `Elasticsearch ICU Analysis`_ plugin
-- RabbitMQ_ v3.5+
+   docker-compose up
 
-.. _PostgreSQL: http://www.postgresql.org/
-.. _Elasticsearch: https://www.elastic.co/
-.. _Elasticsearch ICU Analysis: https://www.elastic.co/guide/en/elasticsearch/plugins/current/analysis-icu.html
-.. _RabbitMQ: https://rabbitmq.com/
+You'll now have some Docker containers running the PostgreSQL, RabbitMQ, and
+Elasticsearch services. You should be able to see them by running ``docker
+ps``. You should also be able to visit your Elasticsearch service by opening
+http://localhost:9200/ in a browser, and connect to your PostgreSQL by
+running ``psql postgresql://postgres@localhost/postgres`` (if you have psql
+installed).
 
-You can install these services however you want, but the easiest way is by using
-Docker and Docker Compose. This should work on any operating system that Docker
-can be installed on:
+Use pyenv to install Python and tox
+-----------------------------------
 
-1. Install Docker and Docker Compose by following the instructions on the
-   `Docker website`_.
+`pyenv`_ installs versions of Python in a ``~/.pyenv`` directory in your home
+directory. It lets you have multiple versions of Python installed at once and
+easily switch between them. This is much easier and safer than using your
+system Python.
 
-2. Run Docker Compose:
+Install Python 2.7 and 3.6 in pyenv and install tox in each:
 
-   .. code-block:: bash
+.. code-block:: shell
 
-      docker-compose up
+   pyenv install 2.7.16
+   pyenv install 3.6.8
+   pyenv shell 2.7.16
+   pip install tox>=3.8.0
+   pyenv shell 3.6.9
+   pip install tox>=3.8.0
+   pyenv shell --unset
```

Comment:
Looks like to do this you do: `pyenv shell system`.

Reply:
`pyenv shell --unset` should switch you back to your system version of Python _except_ if you're in a directory that contains a `.python-version` file, which the `h` directory now contains, then pyenv will automatically use the version(s) of Python listed in  the `.python-version` file whenever you're in that directory. But if you `cd` out of that directory then it will go back to using your system Python